templates/patient_extension.js.erb

```diff
@@ -0,0 +1,152 @@
+  ethnicity: Code,
+  race: Code,
+  sex: Code,
+  given_names: [String],
+  family_name: String,
+  bundle_id: String,
+  // These are the "data criteria", or QDM datatype elements that exist on a
+  // patient.
+  data_elements: []
```

Comment:
i thought this was getting renamed

Reply:
Yeah it did get renamed, there a few comments that reflect the old naming though, and I am addressing that right now.